concepts/05 UI Components/TagBox/00 Getting Started with TagBox/60 Configure Multiple Tags Selection.md

```diff
@@ -0,0 +1,71 @@
+Use the following properties to adapt the TagBox text field to multiple selected tags:        
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)     
+Specifies whether to display tags on multiple lines when the tags overflow the component's width.
+
+- [maxDisplayedTags](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#maxDisplayedTags)      
+Specifies the limit after which the TagBox replaces selected tags with a single multi-tag.
+
+- [showMultiTagOnly](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showMultiTagOnly)      
+Specifies whether to display the multi-tag without ordinary tags. This property is not demonstrated in this tutorial. Refer to the following demo to see it in action: [Tag Count Limitation](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TagBox/TagCountLimitation/).
+
+Drop-down list values can display selection checkboxes. To enable this feature, set the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls) property to **true**. This property also adds the *"Select All"* checkbox to the list.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#tagBox").dxTagBox({
+            // ...
+            multiline: true,
+            maxDisplayedTags: 6,
+            showSelectionControls: true
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-tag-box ...
+        [multiline]="true"
+        [maxDisplayedTags]="6"
+        [showSelectionControls]="true">
+    </dx-tag-box>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxTagBox ...
+            :multiline="true"
+            :max-displayed-tags="6"
+            :show-selection-controls="true"
+        />
+    </template>
+
+    <script>
+        // ...
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    // ...
+    function App() {
+        // ...
+        return (
+            <TagBox ...
+                multiline={true}
+                maxDisplayedTags={6}
+                showSelectionControls={true}
+            />
+        );   
+    }
+
+    export default App;
+    
+---
+
+
+Use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode) property to specify whether to apply values instantly or when users click the *"OK"* button. You can also enable the [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems) property if you want already selected items to be removed from the drop-down list. These properties are not demonstrated in this tutorial.
```

Comment:
Avoidable passive voice and strange word order.

```suggestion
Use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode) property to specify whether to apply values instantly or when users click the *"OK"* button. You can also enable the [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems) property if you want to remove already selected items from the drop-down list. These properties are not demonstrated in this tutorial.
```

Reply:
ok, I apply this change but I've got a question: why do you call this 'strange word order'?

![image](https://user-images.githubusercontent.com/33040934/124268161-d5b99a80-db41-11eb-8a05-b33d090c024c.png)

Reply:
Yes, I called it that way not because English does not allow such things, for it allows many different things. I called this word order "strange" because it sounds like a formal business or, more likely, legal writing style.
Such styles have no place in our documentation because they are not read easily by those who learned English as a second language. We also have to keep in mind that many non English-speaking customers use online translators to understand documentation. This is the main reason why we simplify our texts in terms of tenses, phrases, sentence complexity, etc.